### PR TITLE
On CI redirect google maven repo queries to local mirror for integration test

### PR DESCRIPTION
**Pull request by donat**
This PR resolves https://github.com/gradle/ci-health/issues/136

### CODE REVIEW

**Comment by bmuschko** [comment]
Thanks, LGTM. Did you run a branch build for it so we can see if actually works?

File: subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/RepoScriptBlockUtil.groovy [hunk 1]
```diff
@@ -86,4 +90,13 @@ class RepoScriptBlockUtil {
             """
         }
     }
+
+    static String googleRepositoryDefinition() {
+        String repoUrl = System.getProperty('org.gradle.integtest.mirrors.google')
+        if (repoUrl) {
+            return "maven { name 'Google'; url '${repoUrl}' }"
```
Comment (bmuschko) [inline subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/RepoScriptBlockUtil.groovy]: :o: I guess we could just use a multi-line String. Might make it more readable.

**Comment by bmuschko** [comment]
With the changes from the `release` branch `AndroidSmokeTest` needs to use that now.

File: subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/RepoScriptBlockUtil.groovy (hunk 1 above)
Comment (donat) [reply subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/RepoScriptBlockUtil.groovy]: Fixed

**Pull request merged**